include/revng/Yield/PTML.h

```diff
@@ -58,7 +59,8 @@ std::string functionNameDefinition(const ::ptml::MarkupBuilder &B,
 /// ```
 std::string functionLink(const ::ptml::MarkupBuilder &B,
                          std::string_view Location,
-                         const model::Binary &Binary);
+                         const model::Binary &Binary,
+                         const model::NamingHelper &NamingHelper);
```

Comment:
Not yet, but we definitely planned to.

But honestly, I'd rather wait until we merge revng-c into revng so that we end up with a single set of `ptml::XBuilder`s encapsulating everything. (I feel like I provided this same argument last time we discussed this).

Not much point changing them now considering that not _too_ bad just yet _and_ the whole thing is going to change sooner rather than later.

Reply:
Uff, refactoring never ends. OK.